genrules.cpp

```diff
@@ -3064,14 +3082,20 @@ int Game::GenRules(const AString &rules, const AString &css,
 		temp += "s while they have an unfinished ship in their "
 			"possession, the ship will be discarded and lost. ";
 		temp += "Finally, ships are never interacted with as objects "
-			"directly, but when completed are placed in Fleet "
-			"objects.  Fleets may contain one or more ships, and "
+			"directly, but when completed are placed in Fleets. "
+			"Fleets may contain one or more ships, and "
 			"may be entered like other buildings.";
 		f.Paragraph(temp);
 		temp = "";
 		if (Globals->FACTION_LIMIT_TYPE == GameDefs::FACLIM_FACTION_TYPES) {
-			temp += "Only factions with at least one faction point spent on "
-				"trade can issue ";
+			if (Globals->BUILD_NO_TRADE) {
+				temp = +"Any faction ";
```

Comment:
I guess this was intended as `temp += "Any faction ";`

Reply:
Yes not sure how that compiled that way. I'm pretty sure I checked the output and it was correct... I'll double check it tonight when I get off work.

Reply:
This has been addressed and patch pushed.